frontend/src/App.js

```diff
@@ -16,11 +16,25 @@ class App extends Component {
 
   render() {
     return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
-        </div>
+      <div className="Home">
+        <NavBar />
+        <HomeContent />
+      </div>
+    );
+  }
+}
+
+class HomeContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {isLoggedIn : false};
```

Comment:
nit optional:
maybe make
```
this.state = {
  isLoggedIn: false
}
```
if we plan on adding more states later(?)

Reply:
I'll change it to that when I add more states if that's okay haha

Reply:
yah ofc lol